### DISCUSSION TITLE
Jellyseerr / Overseerr link

### DISCUSSION THREAD

**Discussion by nioc** [jellyfin/jellyfin-web#4818] [Ideas]
Hello, I was wondering if it should be possible that Jellyfin web client can expose a link to a Jellyseerr / Overseer (catalog GUI) ?
This feature should let user in Jellyfin Android app search and request for a content without to open a browser.

**Comment by thornbill** [comment]
You can follow this documentation to add custom links:

https://jellyfin.org/docs/general/clients/web-config#custom-menu-links

Links will open in a browser or separate web view (I'm uncertain about the Android app specifically). Opening random websites directly in the app could break the app or even introduce security vulnerabilities.

**Reply by nioc** [reply]
Thank you for directing me to this feature that I was unaware of.

**Reply by nioc** [reply]
I've just tested this solution and it works like a charm, even if my wife would have preferred a webview instead of a browser opening 😄
Thanks again.
